cmd/snap-confine/mount-support-nvidia.c

```diff
@@ -113,6 +113,7 @@ static const char *nvidia_globs[] = {
 	"libnvidia-ml.so*",
 	"libnvidia-opencl.so*",
 	"libnvidia-ptxjitcompiler.so*",
+	"libnvidia-rtcore.so*",
```

Comment:
Like in https://github.com/snapcore/snapd/pull/8397/files, I believe we also need a corresponding change in opengl.go.

Reply:
https://github.com/snapcore/snapd/blob/df6919d946f42c48f2e388c7446e626178a894a5/interfaces/builtin/opengl.go#L40 Will this wildcard not pick it up?

Reply:
Oh, https://github.com/snapcore/snapd/pull/8419#issuecomment-608503207 pointed out that it is already picked up by this rule:
```
/var/lib/snapd/hostfs/{,usr/}lib{,32,64,x32}/{,@{multiarch}/}libnvidia*.so{,.*} rm,
```

Approving.